README.md

```diff
@@ -68,16 +68,26 @@ To successfully run scripts that require client credentials, you will need to fo
 ### Tooling
 
 - **[Python Guidelines — Creative Commons Open Source][ccospyguide]**
-- [Black][black]: the uncompromising Python code formatter
+- [Black][black]: the uncompromising Python code formatter, by [psf][psf]. It
```

Comment:
You didn't format your links properly
> [psf][psf] 

and 

> [PyCQA][pycqa]

I'm also not sure if the extra description is necessary.